README.md

```diff
@@ -32,51 +33,105 @@ The current table is created by the following method:
+## What is the EDGAR text search tool?
 
-The script will give you a .CSV table with the date, name of the company, place of registration, place of business, ticker symbol, filing type, filing link, and document link. 
+If you're interested in finding all the documents mentioning a certain person or a phrase in the EDGAR database, you can do that via the [full text search page](https://www.sec.gov/edgar/search/#)
 
-Let me know how it works! In a few months I will provide an updated version that automatically breaks over-ambitious searches into manageable time frames, and I will also try to include whatever other reasonable suggestions I receive. 
+Since the SEC does not expose an API endpoint enabling programmatic usage of this functionality, we built a Python tool allowing users to replicate the search behavior of the EDGAR text search page, and to download results to a file.
 
-# RSS Feed Customizer
+The tool takes the form of a CLI (soon available as Python library as well) that takes a search query, spawns a (headless by default) browser, and downloads the search results. 
+Search results are then parsed and saved in a CSV or a JSONLines file.
 
-EDGAR has three RSS feeds, and I wrote a [RSS feed parsing script](EDGAR_RSS_custom.py)  that allows you to filter the feed to return only company/tickers of interest. The output is similar to the text search script in that it returns a .CSV with the name of the company, place of registration, place of business, ticker symbol, filing type, filing link, and document link. Here too, I would be happy to improve the tool according to your proposals. 
+## Features
+
+### Search parameters
+
+Most search parameters from the EDGAR text search page are supported, including:
+- `Document word or phrase` (mandatory)
+- `Company name, ticker or CIK, or individual's name` (optional)
+- `Filing category` (optional)
+- `Filed from` and `Filed to` dates (optional)
+
+Currently unsupported search parameters are:
+- `Filed date ranges` (since the same behavior can be achieved with `Filed from` and `Filed to` dates)
+- `Principal executive offices in` (though it could be added in the future by hardcoding the list of supported values)
+
+### Output formats
 
-# Installation
+Currently supported outputs formats are:
+- CSV
+- JSONLines (one JSON object per line)
 
-## Use a Python Virtual Environment
-This helps manage python package versions, for more information [see here](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments)
+Output format is determined by the file extension of the output file path. 
+Accepted values are `.csv` and `.jsonl` (case-insensitive).
 
-`python -m venv .venv`
+### Browsers
 
-Then on UNIX or Mac, activate the venv with:
+Currently supported browsers are:
+- Chrome (default)
+- Firefox
+- Edge
+- Safari
 
-`source .venv/bin/activate`
+However, maximum stealthiness is achieved with Chrome, and we recommend using it 
```

Comment:
Sure ! Shall I set Company Name / Person Name / Email randomly (e.g. using `faker` library ? Not sure whether it can be useful at all for future users to set this themselves, the less CLI args the better, right ? :)